cmd/ceremony/README.md

```diff
@@ -9,6 +9,7 @@ ceremony --config path/to/config.yml
 `ceremony` operates in one of three modes
```

Comment:
While you're here can you change this from 'three modes' to 'multiple modes' or something, I totally forgot to ever change that.